src/fastoad/openmdao/problem.py

```diff
@@ -63,3 +83,48 @@ def write_outputs(self):
                 VariableList.from_problem(self, promoted_only=True), add_variables=True
             )
             writer.write(variables)
+
+    def read_inputs(self):
+        """
+        Reads inputs of the problem.
+        """
+        if self._metadata is not None and self._metadata["setup_status"] == _SetupStatus.POST_SETUP:
+            self._read_inputs()
+        else:
+            self._read_inputs_after_setup = True
+
+    def _get_problem_inputs(self) -> Tuple[om.IndepVarComp, VariableList]:
```

Comment:
PyCharm helped me to spot this one:
```suggestion
    def _get_problem_inputs(self) -> Tuple[VariableList, VariableList]:
```